app/views/support_interface/staff/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :page_title, 'Staff' %>
+
+<h1 class="govuk-heading-l">Staff</h1>
+
+<p class="govuk-body">
+  <%= govuk_button_link_to 'Invite staff user', new_staff_invitation_path %>
+</p>
+
+<% @staff.find_each do |staff| %>
+  <h2 class="govuk-heading-m"><%= staff.email %></h2>
+
+  <%= govuk_summary_list do |summary_list|
+    summary_list.row do |row|
+      row.key { 'Created at' }
+      row.value { staff.created_at.to_s }
+    end
+
+    if staff.created_by_invite?
+      summary_list.row do |row|
+        row.key { 'Invitation status' }
+
+        row.value do
+          if staff.invitation_accepted?
+             govuk_tag(text: "Accepted", colour: "green")
+          else
+            govuk_tag(text: "Not accepted", colour: "red")
```

Comment:
I think red is more for failed states:

```suggestion
            govuk_tag(text: "Not accepted", colour: "grey")
```

Though maybe red is better to draw attention?

Just bikeshedding, feel free to ignore.

Reply:
Hmm good point, I like that it draws attention as really we should have too many users with open invitations about. I think I'll leave it and see what a designer has to say.